jobs/pull-test-infra-bazel.sh

```diff
@@ -40,18 +40,22 @@ buildables=$(bazel query \
     "kind(.*_binary, rdeps(//..., set(${files[@]})))")
 rc=0
 if [[ ! -z "${buildables}" ]]; then
+  echo "bazel build ${buildables}"
   bazel build ${buildables} && rc=$? || rc=$?
 fi
 
-# Clear test.xml so that we don't pick up old results.
-find -L bazel-testlogs -name 'test.xml' -type f -exec rm '{}' +
+if [[ -e bazel-testlogs ]]; then
+  # Clear test.xml so that we don't pick up old results.
+  find -L bazel-testlogs -name 'test.xml' -type f -exec rm '{}' +
+fi
 
 # Run affected tests.
 if [[ "${rc}" == 0 ]]; then
   tests=$(bazel query \
       --keep_going \
       --noshow_progress \
       "kind(test, rdeps(//..., set(${files[@]}))) except attr('tags', 'manual', //...)")
+  echo "bazel test ${tests} //verify:verify-all"
   bazel test --test_output=errors ${tests} //verify:verify-all && rc=$? || rc=$?
```

Comment:
I think we should always run them. If you change ANY source file then we need to run them, and I think it's safe to assume that a PR changes a source file.

@apelisse was looking into smarter ways to run these, but I'm not sure if anything came up.